Bottom: bf66ee343d3adc7b86e000c0834a0c35148b28e3
Top:    f3a9c5cab63333fe93930c745f56ec5cd7585a09
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 01:16:05 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 9b775ac..d1b0452 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -31,7 +31,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 static int step_set_layer (const char *name, int group, int empty);
 static void use_gc (hidGC gc);
